src/test/groovy/sdkman/specs/PlatformSpec.groovy

```diff
@@ -26,12 +26,13 @@ class PlatformSpec extends SdkmanEnvSpecification {
 		"Linux"  | "armv7l"  | "linuxarm32hf"
 		"Linux"  | "armv8l"  | "linuxarm32hf"
 		"Linux"  | "aarch64" | "linuxarm64"
-		"Linux"  | ""        | "linuxx64"
+		"Linux"  | ""        | "exotic"
 		"Darwin" | "x86_64"  | "darwinx64"
 		"Darwin" | "arm64"   | "darwinarm64"
 		"Darwin" | ""        | "darwinx64"
 		"MSYS64" | "i686"    | "msys64"
 		"MSYS64" | ""        | "msys64"
+    "Linux"  | "ppc64le" | "exotic"
```

Comment:
suggestion: Mind if we respect the ordering of the other entries in this table? This one can come one line above `exotic`. Also, I'd expect to see entries for all the exotic platforms.

Reply:
what are "all" exotic platforms?

Reply:
In my opinion, any platform that is not directly supported is exotic.

Reply:
Sure, but I would expect at least the ones you explicitly mentioned to be tested. In other words:
```
        alpha | i64 | ppc | ppc64le | ppc64el | s390 | s390x)
		echo "exotic"
		;;
```
could each be a line in the table.